tests/Unit/Infrastructure/Event/AuthenticationListenerTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Unit\Infrastructure\Event;
+
+use OpenCFP\Test\WebTestCase;
+
+/**
+ * @coversNothing
```

Comment:
Could you add  ```@covers \OpenCFP\Infrastructure\Event\AuthenticationListener``` instead of ```@coversNothing``` here

Reply:
Sure, sorry. I did not notice that CS fixer adds those.